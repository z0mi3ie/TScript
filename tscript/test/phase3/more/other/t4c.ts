//print = function (x) { console.log(x); };

var x;

//x = "//print = function (x) { console.log(x); };\n";
x = readln();

print(x);

