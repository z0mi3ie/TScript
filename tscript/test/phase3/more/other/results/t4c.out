when the going gets weird

